src/components/dialogmote/motehistorikk/MotehistorikkPanel.tsx

```diff
@@ -1,90 +1,117 @@
 import { DialogmotePanel } from "../../mote/components/DialogmotePanel";
 import { FortidenImage } from "../../../../img/ImageComponents";
 import { FlexRow } from "../../Layout";
-import React, { ReactElement, useState } from "react";
+import React, { ReactElement, ReactNode, useState } from "react";
 import {
   DialogmoteDTO,
   DialogmoteStatus,
+  DocumentComponentDto,
   MotedeltakerVarselType,
 } from "@/data/dialogmote/types/dialogmoteTypes";
-import { tilDatoMedUkedagOgManedNavn } from "@/utils/datoUtils";
+import { tilDatoMedManedNavn } from "@/utils/datoUtils";
 import styled from "styled-components";
 import { Forhandsvisning } from "../Forhandsvisning";
-import { Normaltekst } from "nav-frontend-typografi";
-import { TrackedFlatknapp } from "../../buttons/TrackedFlatknapp";
 import { useDialogmoteReferat } from "@/hooks/dialogmote/useDialogmoteReferat";
+import Lenke from "nav-frontend-lenker";
+import { useTrackOnClick } from "@/data/logging/loggingHooks";
 
 const texts = {
   header: "Møtehistorikk",
   subtitle:
     "Oversikt over tidligere dialogmøter som ble innkalt i Modia (inkluderer ikke historikk fra Arena).",
-  avlystMote: "Avlyst møte",
-  avholdtMote: "Avholdt møte",
+  avlystMote: "Avlysning av møte",
+  avholdtMote: "Referat fra møte",
   referat: "Referat",
   avlysningsBrev: "Avlysningsbrev",
 };
 
-const InlineNormaltekst = styled(Normaltekst)`
-  display: inline;
-`;
+interface ForhandsvisDocumentLenkeProps {
+  document: DocumentComponentDto[];
+  title: string;
+  children: ReactNode;
+}
 
-const FlatknappWithMargin = styled(TrackedFlatknapp)`
-  margin-left: 1em;
-`;
+const ForhandsvisDocumentLenke = ({
+  document,
+  title,
+  children,
+}: ForhandsvisDocumentLenkeProps) => {
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+  const trackOnClick = useTrackOnClick();
+  return (
+    <>
+      <li>
+        <Lenke
+          data-cy={title}
+          href="#0"
+          onClick={() => {
+            trackOnClick(title, texts.header);
+            setModalIsOpen(true);
+          }}
+        >
+          {children}
+        </Lenke>
```

Comment:
En Lenke som ikke er en lenke, men en knapp?

Reply:
Og hvorfor er det en `li`?

Reply:
På en måte ja. Tekst stylet som en lenke, men som åpner en modal. Tenkte det var enkleste måten å gjøre det på uten å måtte bruke vanlig tekst med full lenke-styling, men åpen for bedre forslag. Kan også høre med Espen om det er meningen at den skal se ut som en lenke.

Reply:
`li` fordi det skal være en punktliste

Reply:
Det har jo kanskje noe å si for UU? Ellers er jeg enig i at det kan være greit å slippe å bare kopiere styling!

Reply:
Ja, UU kan være et poeng. Undersøker med Espen!

Reply:
Men det er jo ingenting ved navnet på komponenten som tilsier at det er et listeelement? Det har jo med bruken av komponenten å gjøre?